docs/building-documentation.md

```diff
@@ -0,0 +1,20 @@
+# Building the documentation
+
+So meta. First of, activate your virtualenv and install dedicated dependencies:
+
+```shell
+$ pip install -r requirements/doc.pip
+```
+
+Then you should be able to build and serve the documentation using the [MkDocs][mkdocs] commands:
+
+```shell
+$ mkdocs serve
+```
```

Comment:
Just have the documentation on localy :)
But you're right, it's a details, forget it